src/lib/y2storage/proposal/lvm_helper.rb

```diff
@@ -105,11 +105,12 @@ def reusable_volume_groups(devicegraph)
       #   nil when no volume group will be reused.
       #
       # @see volume_group
+      # FIXME: return value of writer is really strange. It should be same as argument.
```

Comment:
I agree. Maybe we can rename the method as `reuse_volume_group(vg)`

Reply:
in fact see what I write to email to yast-devel. return value is ignored, so in the end, I simply remove return value completely.